testsuite/utils/src/main/resources/META-INF/keycloak-server.json

```diff
@@ -168,6 +188,13 @@
         }
     },
 
+    "realmCache": {
+        "provider": "${keycloak.realm.cache.provider:}",
+        "default" : {
+            "enabled": false
```

Comment:
Is this necessary? We need to do this eventually, but we should probably start in the testsuite so we are sure this doesn't break anything. For example, these changes: https://github.com/keycloak/keycloak/pull/10508 were necessary to make authz work without cache.

Reply:
will revert it, thanks.